common/constants/shared.ts

```diff
@@ -139,7 +139,6 @@ export const LONG_CHART_COLOR = PLOTLY_COLOR[1];
 export const pageStyles: CSS.Properties = {
   float: 'left',
   width: '100%',
```

Comment:
Are these needed anymore. Cause when we removed the mac width the other two css lines became irrelevant.

Reply:
Removed pageStyle, was also used in notebooks so was going to do it separately but removed from both.